ProcessMaker/Console/Commands/RetryScriptTasks.php

```diff
@@ -45,6 +46,9 @@ public function handle()
     {
         $tasks = $this->retrieveTaskList();
 
+   	    $requestIds = $tasks->pluck('process_request_id');
+        ProcessRequestLock::whereIn('process_request_id', $requestIds)->delete();
+
         if (!$tasks->count()) {
```

Comment:
Move this count above the ProcessRequestLock. No need to run ProcessRequestLock if the task->count is zero.